KGP.TicketApp.Backend/Program.cs

```diff
@@ -46,9 +47,23 @@ public static void Main(string[] args)
                     (Encoding.UTF8.GetBytes(builder.Configuration.GetSection("Backend").Get<ApplicationOptions>().JwtKey)),
                     ValidateIssuer = true,
                     ValidateLifetime = true,
-                    ValidateIssuerSigningKey = true
+                    ValidateIssuerSigningKey = true,
+                    AudienceValidator = JwtTokenHelper.ClientTypeValidator
+                };
+            }).AddJwtBearer("Organizer", o =>
```

Comment:
Zróbmy gdzieś, np w tym JwtTokenHelpers, żeby "Organizer" i "Client" to były publiczne stałe

Reply:
to po 18 dodam